src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -133,7 +134,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
 
     $state_for_not_gcp_hosted = [];
     $gceServiceAccountAuth = new GceServiceAccountAuthentication(\Drupal::service('apigee_edge.authentication.oauth_token_storage'));
-    if ($gceServiceAccountAuth->isAvailable()) {
+    $is_gce_hosted_site = FALSE;
+    try {
+      $is_gce_hosted_site = $gceServiceAccountAuth->isAvailable();
+    } catch (Exception) {
```

Comment:
Newline after closing curly brace.

Reply:
![image](https://github.com/apigee/apigee-edge-drupal/assets/75600200/1b2442e9-b522-4cfd-ba5c-2d124cb25bce)

Reply:
Thx for reporting but why this violation is not spotted by GHA? :)

Reply:
In our forked repository it works, but in the 3.x branch the GHA is not checking the PR files,
(we will work on it.)
See https://github.com/shishir-intelli/apigee-edge-drupal/actions/runs/7408909761/job/20158083977

Meanwhile, please setup GHA in your repository as well.

Reply:
> Meanwhile, please setup GHA in your repository as well.

You are right, I need to double check if that is working for not, but for quick hotfixes like this, I just use Github UI and hope that any and every issue on my change will be reported by GHA running on this repo/branch.